.github/workflows/build.yml

```diff
@@ -61,7 +61,7 @@ jobs:
 
       - uses: dtolnay/rust-toolchain@master
         with:
-          toolchain: "1.60"
+          toolchain: "1.61"
```

Comment:
Would be nice to mention the rationale for doing this here/now in the commit message -- is it because aws-lc-rs needs this?

Reply:
rustls-webpki -> ring 0.17 needs it

Reply:
Ah, forgot about that.